setup_support.py

```diff
@@ -361,3 +369,78 @@ def _provide_lean_script_contents(
         setup_kwargs['scripts'] = scripts
 
     return setup_kwargs
+
+
+def get_long_description_from_README():
+    """Read README.md, convert to .rst using pypandoc
+
+    If pypandoc is not available or fails - just output original .md
+    """
+    # PyPI doesn't render markdown yet. Workaround for a sane appearance
+    # https://github.com/pypa/pypi-legacy/issues/148#issuecomment-227757822
```

Comment:
PyPI does render markdown these days and setuptools will alert it to do so [as of 38.6.0](https://setuptools.readthedocs.io/en/latest/setuptools.html#metadata), so you could do something like:

```Python
ret = {}
if setuptools.__version__ > '38.6.0':  # Probably want a more correct check than this
    ret['long_description'] = open(README).read()
    ret['long_description_content_type'] = 'text/markdown'
    return ret

# Convert or fall-back

return {'long_description': long_description}
```

If it's even worth bothering.

Reply:
Sweet, thanks! I will incorporate it

Reply:
done